pallets/capacity/src/types.rs

```diff
@@ -214,9 +216,11 @@ where
 	}
 
 	/// Decrease a target's total available capacity.
+	/// To be called on an unstake.
 	pub fn withdraw(&mut self, capacity_deduction: Balance, tokens_staked_deduction: Balance) {
 		self.total_tokens_staked = self.total_tokens_staked.saturating_sub(tokens_staked_deduction);
 		self.total_capacity_issued = self.total_capacity_issued.saturating_sub(capacity_deduction);
+		self.remaining_capacity = self.remaining_capacity.saturating_sub(capacity_deduction);
```

Comment:
...but not decreasing it here!

Reply:
Interesting. In the demo that I did, the capacity was reduced.

Reply:
I'm almost tempted to go back to reduce here.

Reply:
I remember that.  I will look at the commit history.  I was in that code too but I never messed with any fields, I just renamed, but as you say maybe it was from a rebase problem.